openid-federation-1_0.xml

```diff
@@ -1378,77 +1378,43 @@
               Registration as described in <xref target="explicit"/>,
               then this claim is REQUIRED.
             </t>
-            <t hangText="request_authentication_methods_supported">
+
+            <t hangText="pushed_authentication_methods_supported">
               <vspace/>
               OPTIONAL.
```

Comment:
The whole `pushed_authentication_methods_supported` metadata parameter is currently marked as optional. Would it make sense to recommend (or even require) this metadata parameter for all OIDC OPs that support PAR (or at least: that support PAR for registration as described in Section 12, similar to the `federation_registration_endpoint`)?

Reply:
`pushed_authentication_methods_supported` has been removed.